pkg/file/file.go

```diff
@@ -25,14 +25,26 @@ package file
 
 import (
 	"path/filepath"
+	"strings"
 )
 
-// ExtensionIn returns true if the file's extension is one of the provided ones
-func ExtensionIn(path string, extensions []string) bool {
+// HasExtension returns true if the file's extension is one of the provided ones
+func HasExtension(path string, extensions []string) bool {
 	for _, ext := range extensions {
```

Comment:
there is a [package for search and sort](https://golang.org/pkg/sort/#SearchStrings)

Reply:
Thanks for the info, but in this specific case I do not see the value of having to sort the array, then using sort.SearchStrings to find an index and after having to understand if the index is equal to len(s) to see if it did not exist. 

I find my code much more efficient and readable. Anyway, I will move the call to filepath.Ext() out of the loop to avoid calculating it in every iteration of the loop.